lib/tasks/docs.rake

```diff
@@ -485,6 +487,18 @@ namespace :docs do
     "https://primer.style/view-components/stories/?path=/story/#{path}"
   end
 
+  def lookbook_url(component)
+    path = component.name.underscore
+
+    "https://primer.style/view-components/lookbook/inspect/#{path}/default/"
```

Comment:
Does this mean that there needs to be a preview called "default"?

Reply:
I don't mind what we call it, but we do need to make sure they all have the same method

Reply:
I started an ADR for preview/story format and recommend that every component has a "default" so I like this 😄

Reply:
@jonrohan perhaps we could add a linter or test that makes sure each component has a preview called `default`, then?